includes/class-wc-stripe-customer.php

```diff
@@ -169,11 +171,16 @@ protected function generate_customer_request( $args = [] ) {
 				$billing_last_name = get_user_meta( $user->ID, 'last_name', true );
 			}
 
+			// If the user email is not set, use the billing email.
+			if ( empty( $email ) ) {
```

Comment:
Nit: can we co-locate the code that initialises the email and this check?
```suggestion
			$email = $user->user_email;

			// If the user email is not set, use the billing email.
			if ( empty( $email ) ) {
```

Reply:
Good suggestion, updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4599/commits/8407c06303013e52b442e7638ad11fe5cadd5893